### ISSUE TITLE
toAttachment.convert(element).toJSON()

### ISSUE THREAD

**Issue by moriatuch** [socketlabs/socketlabs-nodejs#1]
Hi,
according to the code 
`toAttachment.convert` should return an attachment instance with "toJSON" function, however we got only the second object and it failed on toJSON call, you can see the example below:
line 50 in file toAttachment.js
```
return new attachment(), {
                name: n,
                content: c,
                contentType: t,
                customHeaders: h,
                filePath: f,
                contentId: i
            };
```

so if we will change the code to run the constructor properly it works like it should be:

```
return new attachment( {
                name: n,
                content: c,
                contentType: t,
                customHeaders: h,
                filePath: f,
                contentId: i
            });

```
Thank you, 
Moria

**Comment by MattReibach** [comment]
Hello Moria,

Thank you for the detailed report. I was able to reproduce and confirm this issue. While we work on publishing a fix, I can suggest a workaround that will allow you to use attachments in your messages.

The `toAttachment.convert` function will check if the "attachment" object being passed in is already an instance of the `attachment` class, and if so it will return the instance that you passed in instead of attempting to return a new `attachment`, which is where the bug you identified is occurring.

So depending on how you are building your message attachments, if you use the `attachment` constructor yourself before adding it to the message attachments array, it should bypass this issue.

Thank you again for the detailed issue report. I will keep you updated once a fix has been released.

Thank you,
Matt Reibach
